packages/app-builder-lib/src/winPackager.ts

```diff
@@ -210,7 +210,7 @@ export class WinPackager extends PlatformPackager<WindowsConfiguration> {
     const cscInfo = await this.cscInfo.value
     if (cscInfo == null) {
       if (this.platformSpecificBuildOptions.sign != null) {
-        await sign(signOptions, this)
+        return await sign(signOptions, this)
```

Comment:
I don't think we want to return here, right?
We still have this at the bottom of this function 
```
return this.doSign({ ...
```

Reply:
@mmaietta  We won't reach there if we enter this scope, because of the `return true` on L#219
Added this here for cases when `sign` might return a falsy value, but looking at its definition, seems like that's never the case.

So I assume the fix here would be to 
1. remove this return, await `sign`
2. remove the `return true` in `cscInfo == null` scope so that we do execute `doSign`
3. add some sort of null check for `cscInfo` on L#226 and beyond

But that (step#2) seems wrong since if`cscInfo == null` we have already tried to sign and wouldn't want to try anymore

I am not familiar with what the code does here so I am hoping you can help frame the right logic. To me, the current state looks correct.